setup.cfg

```diff
@@ -66,7 +66,7 @@ lines_between_types = 1
 skip = dbnd/tasks/__init__.py,dbnd/_core/utils/http/reliable_http_client.py,dbnd/__init__.py,dbnd/_vendor
 skip_glob = __init__.py
 known_first_party = databand,dbnd,dbnd_airflow,dbnd_airflow_contrib,dbnd_airflow_export,dbnd_airflow_operator,dbnd_aws,dbnd_azure,dbnd_databricks,dbnd_docker,dbnd_examples,dbnd_gcp,dbnd_hdfs,dbnd_luigi,dbnd_postgres,dbnd_snowflake,dbnd_tensorflow,dbnd_mlflow,dbnd_qubole,dbnd_spark,dbnd_test_scenarios,test_dbnd,test_dbnd_airflow,test_dbnd_airflow,dbnd_web,targets,tests
-known_third_party = Queue,UserDict,_pytest,airflow,airflow_monitor,apache_beam,argcomplete,attr,azure,backports,boto3,botocore,click,croniter,dag_test_examples,dateutil,decorator,docker,flask,flask_admin,flask_appbuilder,google,hjson,jinja2,kubernetes,luigi,matplotlib,mlflow,mock,more_itertools,msvcrt,mxnet,numpy,packaging,pandas,past,pendulum,pkg_resources,prometheus_client,psycopg2,pyarrow,pygments,pyspark,pytest,pytz,pytzdata,qds_sdk,requests,requests_toolbelt,sagemaker,sagemaker_containers,scipy,scoring_server,setuptools,six,sklearn,snowflake,sqlalchemy,tensorflow,torch,torchvision,tzlocal,yaml
+known_third_party = Queue,UserDict,_pytest,airflow,airflow_monitor,apache_beam,attr,azure,backports,boto3,botocore,bs4,click,croniter,dag_test_examples,dateutil,decorator,docker,flask,flask_admin,flask_appbuilder,google,hjson,jinja2,kubernetes,luigi,matplotlib,mlflow,mock,more_itertools,msvcrt,mxnet,numpy,packaging,pandas,past,pendulum,pkg_resources,prometheus_client,psycopg2,pyarrow,pygments,pyspark,pytest,pytz,pytzdata,qds_sdk,requests,requests_toolbelt,sagemaker,sagemaker_containers,scipy,scoring_server,setuptools,sh,six,sklearn,snowflake,sqlalchemy,tensorflow,test_dbnd_airflow_monitor,torch,torchvision,tzlocal,yaml
```

Comment:
please do not commit this file ( please revert)

Reply:
sure - I have one question though - this file keeps being changed by `pre-commit`. why?

Reply:
also, it causes pre-commit to fail

Reply:
fixd - I removed `seed_isort_config` as we discussed on slack